packages/network/samba/package.mk

```diff
@@ -139,6 +139,8 @@ post_makeinstall_target() {
   rm -rf $INSTALL/usr/bin
   rm -rf $INSTALL/usr/lib/python*
   rm -rf $INSTALL/usr/share/perl*
+  rm -rf $INSTALL/usr/lib64/perl*
+  rmdir $INSTALL/usr/lib64
```

Comment:
remove L143 and make L142 `rm -rf $INSTALL/usr/lib64` instead

Reply:
I did that to avoid removing lib64 if it's not empty, with the reasoning that somehow, in another type of host system, something that shouldn't be deleted would end up here. If that ever happens in the future, the "cannot overwrite non-directory" error would happen again to signal that there is something else there. But I can remove everything if desired.

Reply:
lib64 doesn't exist in the install directory of Ubuntu Samba builds so I'm pretty sure it's safe to remove lib64 as whatever is in there we don't need.